src/libDirectoryService/FinalBlockPostProcessing.cpp

```diff
@@ -575,10 +575,10 @@ bool DirectoryService::ProcessFinalBlockConsensusCore(
   }
 
 #ifdef VC_TEST_FB_SUSPEND_RESPONSE
-  ConsensusCommon::State checkState = m_consensusObject->GetState();
+  ConsensusCommon::State state = m_consensusObject->GetState();
```

Comment:
should it be check state instead?

Reply:
need to rebase. this is unintended broken code.